src/test/regress/expected/function_propagation.out

```diff
@@ -1119,7 +1119,7 @@ SELECT create_distributed_function('func_to_colocate(int)', colocate_with:='tbl_
 SELECT distribution_argument_index, colocationid, force_delegation FROM pg_catalog.pg_dist_object WHERE objid = 'func_to_colocate'::regproc;
  distribution_argument_index | colocationid | force_delegation
 ---------------------------------------------------------------------
-                             |        10003 |
+                             |          163 |
```

Comment:
what caused the change in numbering?

Reply:
I think since we drop and recreate colocation groups now, they have different numbers than before. There is no `ALTER SEQUENCE pg_dist_colocationid_seq ..` query in this test, so the colocation group with this colocation id was created in another test, wasn't dropped and was reused in this test. Now it is probably dropped earlier and recreated.
Which is also a reason to not rely on the exact colocation id in the tests.

Reply:
could be nice to add an ALTER SEQUENCE to prevent future changes